examples/2d/contributors.rs

```diff
@@ -41,6 +43,12 @@ struct Velocity {
     rotation: f32,
 }
 
+enum LoadContributorsError {
```

Comment:
I like to define types close to where they are actually used, especially for something like errors. Can we move this closer to the function that produces this error?

Reply:
Moved it closer to the function